app/services/service_provider_seeder.rb

```diff
@@ -6,6 +6,7 @@ def initialize(rails_env: Rails.env, deploy_env: LoginGov::Hostdata.env)
   end
 
   def run
+    check_for_missing_sps
```

Comment:
I think we want this check to run *after* the update, right?

Reply:
It shouldn't matter - running the update won't change the outcome. The check here is whether or not there's an SP record in the DB that _isn't_ in the YAML file. Merging in the YAML changes will not modify an SP that isn't in the YAML file.

We can still move it to the end if it reads better, let me know :-)

Reply:
nah it doesn't make that big of a difference